Eastern-Research-Group/lew
# Feature/128 update dependencies

## Related Issues:
* #128 

## Main Changes:
* Updated to NodeJS 20.
* Updated to ArcGIS JS API 4.28.
* Updated 3rd party dependencies.

## Steps To Test:
1. Run `npm run test`
2. Verify all tests pass
3. Start the app
4. Compare outputs to the dev site
